index.html

```diff
@@ -318,41 +318,31 @@ <h3 id="unacceptablebehavior">
           <li>Use of coded language (also known as "dog whistles") used to
           rally support for hate groups or to intimidate vulnerable groups.
           </li>
+          <li>Patronizing language or behavior:
+            <ul>
+              <li>Assuming that people or groups need concepts defined or explained to them. 
```

Comment:
I think the "particular" adjective was important here, and I would suggest adding it back ("Assuming that particular people...").  IMO, it makes sense to define concepts; the patronizing aspect comes in when it seems condescending, or as if the speaker believes their audience is inferior due to this knowledge differential.  If I were giving a talk on a highly technical subject, it would make sense to assume that some concepts should be defined or explained; it's problematic if I do that because of irrelevant characteristics of my audience (your next point covers that well) or if I am condescending in presenting it.  (Your third point covers this, too - so alternately, you could simply remove this line, although I think that leaves too much unsaid.)

Reply:
1. I agree that this proposed text leave too much unsaid.  To my mind, the point is not that we should not make assumptions about people or groups.  Making assumptions is an inescapable necessity.  The point is that we should not make such assumptions _based on characteristics that are irrelevant to the work objective_, such as appearance, nationality, gender identity, etc.  For example, if we are trying to hire a programmer, and someone has a BS in computer science, it is completely reasonable to _assume_ that that person knows how to program in at least one programming language, because that is a _relevant_ assumption.  We cannot make such assumptions based on the person's skin color, nationality, gender, or any other characteristic that is _irrelevant_ to programming.  That distinction -- between relevant and irrelevant characteristics -- is central the point that this bullet is trying to make, but not actually stating.  I believe it should be stated explicitly, such as in the proposed text at: https://github.com/w3c/PWETF/pull/238

2. I vehemently disagree verbiage (such as this) that in any way encourages people to avoid defining their jargon.  Failing to define one's jargon creates a _less_ inclusive environment.

Reply:
I left "particular" out because I believe anyone can be patronizing to another person, and moving the section out of microagressions makes that even more necessary. Being patronizing is an action centered around making assumptions.

I agree David that making certain assumptions is unavoidable, especially if you're in a situation where you are speaking to a group or are unfamiliar with your audience. The key is that there are things you can do in every situation to avoid being patronizing. 

Here's the challenge, I am not a fan of making documents like these overly wordy and complex. The CEPC is meant to be a guide, not step-by-step instructions for how to behave. In fact, if we attempted to write a step-by-step guide for how to behave, we'd very likely be buried in comments asking for clarification for the remainder of our natural lives.

I absolutely do not understand your assertion that this prevents explaining jargon, David. At no point does this text suggest what you should or should not say.

Reply:
> I absolutely do not understand your assertion that this prevents explaining jargon, David. At no point does this text suggest what you should or should not say.

The text is under "Unacceptable behavior", so it is telling people what NOT to do.  I.e., it is telling us that a speaker should NOT "[assume] that people or groups need concepts defined or explained to them".  Since a speaker in front of an unfamiliar audience can never really know what the audience knows or doesn't know, the only safe course of action to avoid sounding patronizing would be to NOT define or explain one's concepts.  Thus, in effect that guidance is warning people against defining or explaining their concepts.

To my mind, that guidance is exactly wrong.  A speaker SHOULD "assume that people or groups need concepts defined or explained to them".  I.e., the default should _always_ be to define one's jargon.  It should only be skipped if the speaker has enough _relevant_ information about the audience to know that it can be safely skipped.